instagram/src/components/PostContainer/PostContainer.js

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+import "./PostContainer.css";
+import { CommentSection } from "../CommentSection/CommentSection.js";
+import { Card, CardImg, CardText, CardBody, CardTitle, CardSubtitle, Button } from "reactstrap";
+
+export const PostContainer = (props) => {
```

Comment:
You could use object destructuring here (since props is just an object afterall)

```
export const PostContainer = ({ post }) => {
```

then on line 9, you can reference the variable `post`. ie, `post.map()`